dev-support/Jenkinsfile

```diff
@@ -49,6 +50,8 @@ pipeline {
     // These tests currently have known failures. Once they burn down to 0, remove from here so that new problems will cause a failure.
     TESTS_FILTER = 'cc,checkstyle,javac,javadoc,pylint,shellcheck,whitespace,perlcritic,ruby-lint,rubocop,mvnsite'
     EXCLUDE_TESTS_URL = "${JENKINS_URL}/job/HBase-Find-Flaky-Tests/job/${BRANCH_NAME}/lastSuccessfulBuild/artifact/excludes"
+    SHALLOW_CHECKS = 'all,-unit,-findbugs' // run by the 'yetus general check'
+    DEEP_CHECKS = 'maven,mvninstall,compile,javac,unit,htmlout' // run by 'yetus jdkX HadoopY checks'
```

Comment:
I believe we produce different shaded jars with different hadoop versions. is it worth pulling that check over into the hadoop specific builds? it shouldn't vary by jdk version.

Reply:
You're the expert in this area, I'll do as you suggest. That one is just called `shadedjars` ?

Reply:
yes shadedjars is correct.